skimage/measure/tests/test_regionprops.py

```diff
@@ -580,12 +579,22 @@ def test_regionprops_table():
 def test_regionprops_table_equal_to_original():
     regions = regionprops(SAMPLE, INTENSITY_FLOAT_SAMPLE)
     out_table = regionprops_table(SAMPLE, INTENSITY_FLOAT_SAMPLE,
-                                  properties=COL_DTYPES.keys())
-
-    for prop in COL_DTYPES.keys():
-        for i in range(len(regions)):
-            if not isinstance(regions[i][prop], Iterable):
-                assert regions[i][prop] == out_table[prop][i]
+                                  properties=COL_DTYPES.keys(),
+                                  separator="-")
```

Comment:
```suggestion
                                  properties=COL_DTYPES.keys())
```

isn't `"-"` the default value for the `separator` argument?

Reply:
Exactly I noticed it is, thank you for pointing this out!